docs/core/diagnostics/dotnet-trace.md

```diff
@@ -329,6 +329,31 @@ dotnet-trace collect --process-id <PID> --providers System.Runtime:0:1:EventCoun
 
 The preceding command disables runtime events and the managed stack profiler.
 
+## Use .rsp file to avoid typing long commands
+
+`dotnet-trace` supports being launched with a `.rsp` file that contains the arguments to be passed. This can be useful when enabling providers that expect lengthy arguments or when using different shell environment strips different arguments.
```

Comment:
```suggestion
You can launch `dotnet-trace` with an `.rsp` file that contains the arguments to pass. This can be useful when enabling providers that expect lengthy arguments or when using a shell environment that strips characters.
```